l10n_it_declaration_of_intent/views/declaration_of_intent_view.xml

```diff
@@ -0,0 +1,197 @@
+                                <tree editable="top">
+                                    <field name="invoice_id" />
+                                    <field name="date_invoice" />
+                                    <field name="taxes_ids" widget="many2many_tags" />
+                                    <field
+                                        name="base_amount"
+                                        string="Document Amount"
+                                    />
+                                    <field name="amount" />
+                                </tree>
+                            </field>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="declaration_of_intent_search">
+        <field name="name">Search view for declaration_of_intent</field>
+        <field name="model">l10n_it_declaration_of_intent.declaration</field>
+        <field name="arch" type="xml">
+            <search string="Declarations of intent">
+                <field name="type" />
+                <field name="number" />
+                <field name="partner_document_number" />
+                <field name="date" />
+                <field name="partner_document_date" />
+                <field name="partner_id" />
+                <field name="date_start" />
+                <field name="date_end" />
+                <filter
+                    name="state_valid"
+                    string="Valid"
+                    domain="[('state','=','valid')]"
+                />
+                <filter
+                    name="state_close"
+                    string="Close"
+                    domain="[('state','=','close')]"
+                />
+                <filter
+                    string="Partner"
+                    name="partner_group"
+                    context="{'group_by':'partner_id'}"
+                />
+                <filter
+                    string="State"
+                    name="state_group"
+                    context="{'group_by':'state'}"
+                />
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="declaration_of_intent_tree">
+        <field name="name">Tree view for declaration_of_intent</field>
+        <field name="model">l10n_it_declaration_of_intent.declaration</field>
+        <field name="arch" type="xml">
+            <tree
+                decoration-info="state == 'valid'"
+                decoration-muted="state == 'close'"
+                decoration-warning="state == 'expired'"
+            >
+                <field name="type" />
+                <field name="number" />
+                <field name="date" />
+                <field name="partner_id" />
+                <field name="date_start" />
+                <field name="date_end" />
+                <field name="available_amount" />
```

Comment:
anche qui, serve visualizzare currency_id?

Reply:
Da risolvere in https://github.com/OCA/l10n-italy/issues/2428